test/cpu_fusion.cpp

```diff
@@ -1447,34 +1447,69 @@ TEST(cpu_fusion, backwards_maxpool_with_indices_n4_c1_hw4_2x2_max)
     ASSERT_TRUE(read_vector<float>(output) == expected);
 }
 
-#if 0
-TEST(cpu_fusion, loop_kernel_one_input_one_output)
+#if defined(NGRAPH_HALIDE)
```

Comment:
Why don't we keep the halide tests in a halide-specific file.

Reply:
If they are in a separate file then you can use cmake to add the test source file or not depending on halide/no halide